release/release-runner

```diff
@@ -60,6 +60,39 @@ message()
     echo "release-runner: $@" >&2
 }
 
+# check if we don't have a proper home directory, but a secrets volume mounted;
+# if so, initialize HOME=/home/user
+init_container_home()
+{
+    local secrets_dir=/run/secrets/release
+    local home_dir='/home/user'
+
+    [ -d "$secrets_dir" ] && [ -d "$home_dir" ] || return 0
```

Comment:
This isn't 100% clear to me, doesn't this clash with the comment above? This will only continue if both the `secrets_dir` and `home_dir` exist.

Or is `home_dir` assumed to be always present but it needs to be initialized from the secrets dir?

Reply:
Indeed I went back and forth with this a bit, so the comment and the code now don't coincide. /home/user is created in Dockerfile, but it's empty. I'll update the comment.

Reply:
Pushed.